scripts/remember.coffee

```diff
@@ -76,11 +76,12 @@ module.exports = (robot) ->
 
     msg.send "My favorite memories are:\n#{sortedMemories[0..20].join(', ')}"
 
-  robot.respond /(me|random memory)(\s+.+)?$/i, (msg) ->
+  robot.respond /(me|random memory)(\s+.*)?$/i, (msg) ->
     msg.finish()
 
-    randomKey = if msg.match[2]
-      msg.random(findSimilarMemories(msg.match[2].trim()))
+    search = msg.match[2].trim()
```

Comment:
I haven't run this - but is `msg.match[2]` defined when the message is just `.me`? I want to make sure this doesn't bounce off an `undefined` dereference. 

Maybe `= msg.match[2] and msg.match[2].trim()` - which I think can be spelled `= msg.match[2]?.trim()`

Reply:
Good point. Looks like other places in the code do stuff like `msg.match[2]?.trim()`, so seems reasonable to do the same here.